### PR TITLE
Hn.manage validator screen

### LINKED ISSUE #82: Validator management update screens
We have to add additional screens to validator management from Figma.

### PR DESCRIPTION
**Pull request by HristoNedelkov**
Closes status-im#82

Screenshots of the improvements I have made ->
image
image

### CODE REVIEW

**Review by EmilIvanichkovv — CHANGES_REQUESTED** [review]
Please follow the [convention](https://github.com/angular/angular/blob/main/CONTRIBUTING.md#-commit-message-format) we discussed for the commits. Also, I suggest squashing commits that are related to one another and can be combined, because now the history looks a bit busy to me.

Also, can you add a description of the PR with screenshots of the changes it introduces

File: src/pages/ValidatorManagement/ManageValidator/DepositStepper.tsx [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+const DepositStepper = ({ activeStep }: DepositStepperProps) => {
+  const navigate = useNavigate()
+  const windowSize = useWindowSize()
+
+  const stepToUrlMap = [
+    '/validator-management/',
+    // fix paths
+  ]
+
+  const getIsStepVisible = (
```
Comment (EmilIvanichkovv) [inline src/pages/ValidatorManagement/ManageValidator/DepositStepper.tsx:19]: Can you clarify the purpose of this?

Comment (HristoNedelkov) [reply src/pages/ValidatorManagement/ManageValidator/DepositStepper.tsx:19]: @EmilIvanichkovv As you can see I have left a comment that I have to fix it. In the last meet we discussed with Aqeel that I have to wait for further instructions. I still don't know how this stepper should work.

Comment (EmilIvanichkovv) [reply src/pages/ValidatorManagement/ManageValidator/DepositStepper.tsx:19]: I am referring to `getIsStepVisible`. The “math” in there seems kinda odd.

**Review by EmilIvanichkovv — APPROVED** [review]

**Pull request merged**